### PR TITLE
Adding test for checkout: cannot purchase product with no payment CORE_0113

### PR DESCRIPTION
**Pull request by yellowee**
I want to merge this change because it adds a test for checkout: cannot purchase product with no payment CORE_0113

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are either absent or guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migrations are either absent or optimized for zero downtime
* [ ] The changes are covered by test cases

### CODE REVIEW

**Review by fowczarek — APPROVED** [review]

File: saleor/tests/e2e/channel/utils.py [hunk 1]
```diff
@@ -34,6 +34,7 @@ def create_channel(
     currency="USD",
     country="US",
     is_active=True,
+    allowUnpaidOrders=False,
```
Comment (korycins) [inline saleor/tests/e2e/channel/utils.py:37]: Just to keep a consistency, I think you should have `allow_unpaid_orders`. (see format of the rest of arguments)

Comment (korycins) [reply saleor/tests/e2e/channel/utils.py:37]: I don't see usage of the new argument. Is it expected?

Comment (yellowee) [reply saleor/tests/e2e/channel/utils.py:37]: @korycins https://github.com/saleor/saleor/pull/13551/commits/df70a2b98275bad73aab994935dc3d6689deb335

Comment (korycins) [reply saleor/tests/e2e/channel/utils.py:37]: Yeah, but you don't do anything with the `allowUnpaidOrders` that you received in the `create_channel` function. 
Additionally IMO, `allowUnpaidOrders` should be renamed to `allow_unpaid_orders` (I mean the name of argument for function)

Comment (yellowee) [reply saleor/tests/e2e/channel/utils.py]: https://github.com/saleor/saleor/pull/13551/commits/20c95a2d68d1afc9874372ff5971a08ba0f8d519

**Review by korycins — APPROVED** [review]

**Pull request merged**